lib/include/ultrahdr/jpegr.h

```diff
@@ -137,7 +136,7 @@ class JpegR {
  public:
    JpegR();
 
-   JpegR(size_t, int, bool);
+   JpegR(size_t, int, bool, float);
```

Comment:
all constructors can be merged with 
  JpegR(size_t mapDimensionScaleFactor = kMapDimensionScaleFactorDefault,
        int mapCompressQuality = kMapCompressQualityDefault,
        bool useMultiChannelGainMap = kUseMultiChannelGainMapDefault,
        float gainMapGamma = kGainMapGammaDefault);